PREP_SYSTEM_FOR_DIETPI.sh

```diff
@@ -1545,12 +1520,13 @@ _EOF_
 		G_GITOWNER=$gitowner_temp
 		G_GITBRANCH=$gitbranch_temp
 
-		#	reduce sub_version by 1, allows us to create image, prior to release and patch if needed.
-		G_DIETPI_VERSION_SUB=$(( $G_DIETPI_VERSION_SUB - 1 ))
-
 		G_CONFIG_INJECT 'DEV_GITBRANCH=' "DEV_GITBRANCH=$G_GITBRANCH" /DietPi/dietpi.txt
+		G_CONFIG_INJECT 'DEV_GITOWNER=' "DEV_GITOWNER=$G_GITOWNER" /DietPi/dietpi.txt
 		G_VERSIONDB_SAVE
 
+		#	reduce sub_version by 1, allows us to create image, prior to release and patch if needed.
+		G_DIETPI_VERSION_SUB=$(( $G_DIETPI_VERSION_SUB - 1 ))
```

Comment:
@MichaIng 

This needs to be before ```G_VERSIONDB_SAVE``` to ensure lower version is saved?

Reply:
@Fourdee 
I anyway do not use this. I create images always with current master, also to avoid then unnecessary 1st run update from dev/testing => master, in case someone grabs an image with current version at least. Also this would avoid possible issues with current dev/testing code to be written onto an official image, even that the update should mostly solve those.

So since I then revert the lowered version, personally I would remove this completely. But since I just create VM images so far, it should be alone your decision and about your image creation schedule/procedure.

Ah now I see the issue. My reordering broke that, whoopsie. Will revert.